test/nbrowser/WebhookPage.ts

```diff
@@ -52,10 +52,11 @@ describe('WebhookPage', function () {
       'Name',
       'Memo',
       'Event Types',
-      'URL',
       'Table',
-      'Ready Column',
       'Filter for changes in these columns (semicolon-separated ids)',
+      'Ready Column',
```

Comment:
Can you add/update one of this test to update the Authorization column and check that it is persisted ok.